tests/install-darwin.sh

```diff
@@ -52,12 +52,6 @@ function finish {
 trap finish EXIT
 
 # First setup Nix
-cleanup
-curl -o install https://nixos.org/nix/install
-yes | bash ./install
-verify
-
-
```

Comment:
Do the tests work without this?

Reply:
The travisci tests don't work with it, either :( not sure why that broke. I haven't tested this new installer yet on MacOS just yet, solving this will be part of that.